client/modules/IDE/pages/IDEView.jsx

```diff
@@ -212,7 +212,7 @@ class IDEView extends React.Component {
         </Helmet>
         {this.props.toast.isVisible && <Toast />}
         <Nav
-          warnIfUnsavedChanges={this.warnIfUnsavedChanges}
+          warnIfUnsavedChanges={() => warnIfUnsavedChanges(this.props)}
```

Comment:
According to the React docs, you want to creating anonymous functions within render (I know there are many instances of this within the app, and I think they should all be fixed). Instead, I recommend adding a new named function to the IDEView class.

Reply:
Hey Cassie! I've replaced the anonymous function with a named function. Thoughts?

Reply:
Also Cassie, would you advice against using the .bind as well ?